exporter/datadogexporter/internal/logs/sender.go

```diff
@@ -68,6 +65,10 @@ func (s *Sender) SubmitLogs(ctx context.Context, payload []datadogV2.HTTPLogItem
 		s.logger.Debug("Submitting logs", zap.Any("payload", payload))
 	}
 
+	// enable sending gzip
+	// Get a fresh opts for each request to avoid duplicating ddtags
+	s.opts = *datadogV2.NewSubmitLogOptionalParameters().WithContentEncoding(datadogV2.CONTENTENCODING_GZIP)
```

Comment:
@liustanley @sbracegirdle do we need `opts` on sender , since we are initializing this on every request ?

Reply:
That's true, I didn't think of that. I think it makes sense to replace `s.opts` with a local variable in this function.

Reply:
Good idea, i've moved `opts` to a local variable and re-tested.